### PR TITLE
Searching disables the filters

### PR DESCRIPTION
**Pull request by Velin92**
https://github.com/element-hq/element-x-ios/assets/34335419/56bc3594-4043-4e88-a5fc-84a8e4a0de06

### CODE REVIEW

File: ElementX/Sources/Screens/EmojiPickerScreen/View/EmojiPickerScreen.swift
Comment (pixlwave) [inline ElementX/Sources/Screens/EmojiPickerScreen/View/EmojiPickerScreen.swift:1]: ha, could have copy pasted it in the end 😅

File: ElementX/Sources/Mocks/RoomSummaryProviderMock.swift [hunk 1]
```diff
@@ -50,15 +50,22 @@ extension RoomSummaryProviderMock {
...
                 
-                if let query = predicate.query, !query.isEmpty {
-                    rooms = rooms.filter { $0.name?.localizedCaseInsensitiveContains(query) ?? false }
+                roomListSubject.send(rooms)
+            case let .all(filters):
+                var rooms = initialRooms
+                
+                if filters.count > 1 {
+                    // for testing purpose chaining more than one filter will always return an empty state
+                    rooms = []
```
Comment (pixlwave) [inline ElementX/Sources/Mocks/RoomSummaryProviderMock.swift:66]: Double checking that you want to commit this?

**Review by pixlwave — APPROVED** [review]
LGTM 👍

File: ElementX/Sources/Mocks/RoomSummaryProviderMock.swift (hunk 1 above)
Comment (Velin92) [reply ElementX/Sources/Mocks/RoomSummaryProviderMock.swift:66]: yeah is to allow for a new test case, without overcomplicating the mock.

**Pull request merged**